src/components/Admin/resources/competition/problems/createProblemFormData.ts

```diff
@@ -0,0 +1,22 @@
+// approach based on: https://marmelab.com/react-admin/DataProviders.html#handling-file-uploads
+
+import {Problem} from '@/types/api/competition'
+
+// takisto urobene zmeny v RA dataProvideri
+export const createProblemFormData = ({
+  image,
+  solution_pdf,
+  ...data
+}: Omit<Problem, 'image' | 'solution'> & {
+  image?: {rawFile: File}
+  solution_pdf?: {rawFile: File}
+}) => {
+  const formData = new FormData()
+  formData.append('image', image?.rawFile ?? '')
+  solution_pdf?.rawFile && formData.append('solution_pdf', solution_pdf.rawFile)
```

Comment:
preco to su dve rozne pristupy ako to appendnut?

Reply:
noo jeden som prepisal a druhy nie.. a teda solution_pdf fieldy/inputy som nespravil, tak asi este dorobim, no